modules/tinymce/src/themes/silver/test/ts/browser/window/SilverDialogBlockTest.ts

```diff
@@ -0,0 +1,149 @@
+import { Chain, Log, Mouse, Pipeline } from '@ephox/agar';
+import { TestHelpers } from '@ephox/alloy';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Fun } from '@ephox/katamari';
+import { TinyLoader, TinyUi } from '@ephox/mcagar';
+import { Attribute, Height, SelectorFind, SugarElement, SugarLocation, Width } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import { Dialog } from 'tinymce/core/api/ui/Ui';
+import { WindowParams } from 'tinymce/core/api/WindowManager';
+import Theme from 'tinymce/themes/silver/Theme';
+
+import * as DialogUtils from '../../module/DialogUtils';
+
+UnitTest.asyncTest('browser.silver.window.SilverDialogBlockTest', (success, failure) => {
```

Comment:
This test name is missing `tinymce` and `themes`, so it should be this:
```suggestion
UnitTest.asyncTest('browser.tinymce.themes.silver.window.SilverDialogBlockTest', (success, failure) => {
```

Reply:
done